services/movie.py

```diff
@@ -0,0 +1,29 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids=None, actors_ids=None):
+    if genres_ids is None and actors_ids is None:
+        return Movie.objects.all()
```

Comment:
as per video, we need to use some variable(queryset), and later we need to filter the queryset. However, I don't know how it will influence the code.